src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -81,117 +112,278 @@ public void focus() {
+            Text afterLink1 = new Text("\n\n2. Download the latest ");
+            Hyperlink jarLink = new Hyperlink("https://github.com/se-edu/addressbook-level3/releases");
+            jarLink.setOnAction(e -> openHyperlink("https://github.com/se-edu/addressbook-level3/releases"));
 
-            ### Editing the data file
+            Text instructions = new Text("\n\n3. Copy the `.jar` file to your desired folder and open a terminal." +
+                    "\n\n4. Run the application using `java -jar fart_in_a.jar`." +
+                    "\n\n5. Type commands in the command box and press Enter.");
 
-            FART data is saved automatically as a JSON file `[JAR file location]/data/addressbook.json`.
+            helpContentFlow.getChildren().addAll(beforeLink1, javaLink, afterLink1, jarLink);
 
-            ## Command summary
+            processTextWithBackticks(instructions.getText());
+        } else {
+            processTextWithBackticks(bodyText);
+        }
+    }
 
-            Action | Format, Examples
-            --------|------------------
-            **Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/BIRTHDAY [t/TAG]…​` <br>
-             e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123,
-             Clementi Rd, 1234665 b/11 09 2001 t/friend t/colleague`
-            **Clear** | `clear`
-            **Delete** | `delete INDEX`<br> e.g., `delete 3`
-            **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br>
-             e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-            **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-            **List** | `list`
-            **Help** | `help`
-            **Paid** | `paid 3`
-            **Unpaid** | `unpaid 3`
-            """;
+    /**
+     * Processes text and applies formatting to sections within backticks (`` ` ``), treating them as code.
+     */
+    private void processTextWithBackticks(String text) {
+        String[] segments = text.split("`");
+        boolean isCode = false;
+
+        for (String segment : segments) {
+            Text t = new Text(segment);
+            if (isCode) {
+                t.setFont(Font.font("Monospaced"));
+                t.setStyle("-fx-font-size: 14px; -fx-fill: #3b82f6;");
+            } else {
+                t.setStyle("-fx-font-size: 14px;");
+            }
+            helpContentFlow.getChildren().add(t);
+            isCode = !isCode;
+        }
+    }
 
-        // Set the content to the TextArea in the help window
-        helpContent.setText(userGuideContent);
-        helpContent.setWrapText(true);
-        helpContent.setEditable(false);
+    /**
+     * Opens a hyperlink in the default browser.
+     */
+    private void openHyperlink(String url) {
+        try {
+            if (Desktop.isDesktopSupported()) {
+                Desktop.getDesktop().browse(new URI(url));
+            } else {
```

Comment:
As per v1.3 I believe the else branch will always be taken? is there a need for the if branch in this case

Reply:
Could I clarify what you mean by this, as this is meant to check if the user's device is able to support desktop browsing to access hyperlinks?

Reply:
ah i see cause i remember there is an evolved version of this called AB4 which supports browsing from within the app itself
but if you wrote this code, then lgtm